plugins/Monolog/Formatter/LineMessageFormatter.php

```diff
@@ -43,6 +48,21 @@ public function format(array $record)
 
         $message = trim($record['message']);
 
+        if (gettype($this->excludePatterns) === "array") {
+            foreach ($this->excludePatterns as $p) {
+                if (strpos($message, $p) !== false) {
+                    return;
+                }
+                if (strpos($class, $p) !== false) {
+                    return;
+                }
+            }
+        }
+
+        if ($this->logMessageFormat == 'json') {
```

Comment:
might be a tad safer
```suggestion
        if ('json' === $this->logMessageFormat) {
```

Reply:
I agree about the === but is there a reason to put json first?  legibility?

Reply:
It's the [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions) style, in general preventing situations when someone forgets an equal sign and accidentally assigns the value to the variable instead of doing a comparison. With the string or number first that can't happen. It's not a strict thing we check for in CS yet but might be at some point.

Reply:
@michalkleiner hmm..... first time I've seen this, it is.  Modify the line, I will